test/currencyConvertor.ts

```diff
@@ -1,72 +1,89 @@
 import { ethers, waffle } from 'hardhat'
+
+
 import chai from "chai";
 import { solidity } from "ethereum-waffle";
 import CurrencyConvertorArtifact from '../artifacts/contracts/proxies/CurrencyConvertor.sol/CurrencyConvertor.json';
+import StarkwareArtifact from '../artifacts/contracts/exchange-wrappers/Starkware.sol/MockStarkware.json';
 import ZeroExExchangeWrapperArtifact from '../artifacts/contracts/exchange-wrappers/ZeroExExchangeWrapper.sol/ZeroExExchangeWrapper.json';
-import StarkwareArtifact from '../artifacts/contracts/exchange-wrappers/Starkware.sol/Starkware.json';
-import UsdcContractArtifact from '../artifacts/contracts/tokens/mockUsdcContract.sol/MockUsdcContract.json';
+import fakeUSDCContractArtifact from '../artifacts/contracts/tokens/mockUsdcContract.sol/MockUsdcContract.json';
+import { createUser, generateQueryPath } from './helpers';
+import { axiosRequest, RequestMethod } from '@dydxprotocol/v3-client/build/src/lib/axios';
 
 const { deployContract } = waffle
 
-
 chai.use(solidity);
 const { expect } = chai;
+
 describe("CurrencyConvertor", () => {
   let currencyConvertor: any;
   let zeroExExchangeWrapper: any;
   let starkware: any;
-  let usdc: any;
+  let fakeUSDC: any;
+  let user: any;
   beforeEach(async () => {
     const signers = await ethers.getSigners();
 
     zeroExExchangeWrapper = await deployContract(
       signers[0],
       ZeroExExchangeWrapperArtifact,
     );
+    fakeUSDC = await deployContract(
+      signers[0],
+      fakeUSDCContractArtifact,
+    )
```

Comment:
Why are we still deploying a mock USDC contract if we're running tests on a forked version of mainnet? Can you just use https://hardhat.org/hardhat-network/reference/#hardhat-impersonateaccount to impersonate an account with some USDC?